doc/user/content/console/sql-shell.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "SQL Shell"
+description: "SQL Shell in the Materialize console"
+disable_toc: true
+menu:
+  main:
+    parent: console
+    weight: 10
+    identifier: console-sql-shell
+---
+
+The [Materialize Console](https://console.materialize.com/) provides a **SQL
+Shell**, where you can issue your queries. Materialize follows the SQL standard
+(SQL-92) implementation, and strives for compatibility with the PostgreSQL
```

Comment:
I'm not sure how useful / true it is to say we follow the standard. It might be more useful to say that we are postgres compatible?